src/components/Button.tsx

```diff
@@ -16,7 +16,7 @@ export default class Button extends React.Component<ButtonProps, {}> {
 
   render(): JSX.Element {
    let content = this.props.content || "Submit";
-   let className = `btn ${this.props.className || "btn-default"}`;
+   let className = `btn${this.props.className ? ` ${this.props.className}` : ""}`;
```

Comment:
What happens to `btn-default` now?

Reply:
Not needed; the default styling comes from `btn`.